src/encoded/types/treatment.py

```diff
@@ -65,7 +65,8 @@ def display_title(self, request, treatment_type=None, chemical=None,
             conditions = " (" + "".join(d_t) + ")"
 
         if chemical:
-            dis_tit = chemical + " treatment" + conditions
+            word = ' treatment' if concentration != 0 else ' washout'
+            dis_tit = chemical + word + conditions
```

Comment:
I can't speak to whether "washout" is the right word here, but I'll assume that's a domain issue you have better understanding of than I do. It's clear what you're trying to do and functionally it seems fine.

While you're in there, though, I wouldn't mind seeing the variable `dis_tit` renamed to just `title` both for clarity and to reduce awkwardness of the name. I'd also probably rename `word` to `action` and change `chemical + word + conditions` to `chemical + " " + action + conditions` and change the `action` to not have a leading space in its value.

Optionally also change the setup for `conditions` (lines 54 to 65) to eliminate the `d_t` variable and just do something like this (untested, but I think the existing unit tests seem to be adequate--except there was no test of the weird case of no concentration or duration but still a temperature that leads to odd formatting in the existing code, hopefully fixed here):
```
        conditions = ""
        if concentration and concentration_units:
            conditions += str(concentration) + ' ' + concentration_units
        if duration and duration_units:
            if conditions:
                conditions += ', '
            conditions += str(duration) + duration_units[0]
        if temperature:
            if conditions:
                conditions += " "
            conditions += "at " + str(temperature) + "°C"
        if conditions:
            conditions = " (" + conditions + ")"
```

Reply:
For the record, I'm not the one who originally created that variable title... but I'm happy to change it.

The comments about word vs action and placement of the space seem pretty trivial to me, but I can implement them if you think it makes the code read more clearly.